### ISSUE TITLE
OPK SSH should detect OpenSSH version

### ISSUE THREAD

**Issue by EthanHeilman** [openpubkey/openpubkey#215]
OPK only works with `OpenSSH v. 8.1 or greater` . The OPK SSH cli should detect if it is being asked to verify on a server with versions less than 8.1 and produce a warning.

OpenSSH used to impose a 4096-octet limit on the string buffers available to the percent_expand function. In October 2019 as part of the 8.1 release, that limit was removed. If you exceeded this amount it would fail with `fatal: percent_expand: string too long`

The OPK SSH verifier when called by by OpenSSH server using the command `./opkssh ver ...` should determine the version of OpenSSH server  calling it and then log this information to the openpubkey log using `log(fmt.Sprint("OpenPubkey SSH requires OpenSSH v. 8.1 or greater"))`

The path into `./opkssh ver ...` can be found in:https://github.com/openpubkey/openpubkey/blob/main/examples/ssh/opkssh/cli.go#L68

**Comment by SaahilNotSahil** [comment]
@EthanHeilman Does this issue still need to be worked upon?

**Comment by EthanHeilman** [comment]
@SaahilNotSahil

Yes, I never added this feature

**Comment by SaahilNotSahil** [comment]
Sure!

**Comment by SaahilNotSahil** [comment]
@EthanHeilman This is only supposed to be a warning and not an error, right? If the version is less than 8.1, then the program should continue execution and not panic/exit?

**Comment by EthanHeilman** [comment]
@SaahilNotSahil 
Yes, it should just be a warning. It is useful information to have in the log file when debugging a opkssh configured server over email, but opkssh does and should work on versions less than 8.1.

Now that I think about it, in addition to the warning, we should probably always log the openssh version being used.
